src/WebSocketsClient.cpp

```diff
@@ -117,7 +133,9 @@ void WebSocketsClient::loop(void) {
                 _client.tcp = NULL;
             }
             _client.ssl = new WiFiClientSecure();
+#if !defined(ESP32)
             _client.tcp = _client.ssl;

```

Comment:
Intested both websocket server and also websocket client using https://github.com/bbx10/arduinoWebSockets/tree/esp32
I see slower performance compare with ESP8266 with same code and binary frame. Shows intermittent pause inreceiving data from ESP32 to both server and client.